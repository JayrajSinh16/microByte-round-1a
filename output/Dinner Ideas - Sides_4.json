{
  "title": "o 1 teaspoon salt o Skewers  Instructions :",
  "outline": [
    {
      "level": "H1",
      "text": " Ingredients :",
      "page": 1
    },
    {
      "level": "H2",
      "text": " Ingredients :",
      "page": 1
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 1
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 2
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 2
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 2
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 3
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 3
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 3
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 4
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 4
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 4
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 5
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 5
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 5
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 6
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 6
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 7
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 7
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 7
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 8
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 8
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 8
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 9
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 9
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 10
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 10
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 10
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 11
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 11
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 12
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 12
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 12
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 13
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 13
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 13
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 14
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 14
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 15
    },
    {
      "level": "H3",
      "text": " Ingredients :",
      "page": 15
    }
  ]
}