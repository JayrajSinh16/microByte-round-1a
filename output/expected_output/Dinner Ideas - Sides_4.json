{
    "title": "Dinner - Sides",
    "outline": [
        {
            "level": "H1",
            "text": "Pajeon (Pancakes)",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 1
        },
        {
            "level": "H1",
            "text": "Pan con Tomate",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 1
        },
        {
            "level": "H1",
            "text": "Paneer Tikka",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 1
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 2
        },
        {
            "level": "H1",
            "text": "Panzanella",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 2
        },
        {
            "level": "H1",
            "text": "Papadum",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 2
        },
        {
            "level": "H1",
            "text": "Patatas Bravas",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 2
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 3
        },
        {
            "level": "H1",
            "text": "Pepper Pot",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 3
        },
        {
            "level": "H1",
            "text": "Peri-Peri Chicken",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 3
        },
        {
            "level": "H1",
            "text": "Pico de Gallo",
            "page": 3
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 4
        },
        {
            "level": "H1",
            "text": "Pimientos de Padrón",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 4
        },
        {
            "level": "H1",
            "text": "Plantain Chips",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 4
        },
        {
            "level": "H1",
            "text": "Polenta",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 4
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 5
        },
        {
            "level": "H1",
            "text": "Potato Salad",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 5
        },
        {
            "level": "H1",
            "text": "Pretzels",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 5
        },
        {
            "level": "H1",
            "text": "Pulao",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 5
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 6
        },
        {
            "level": "H1",
            "text": "Quesadillas",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 6
        },
        {
            "level": "H1",
            "text": "Quiche Lorraine",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 6
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 7
        },
        {
            "level": "H1",
            "text": "Quindim",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 7
        },
        {
            "level": "H1",
            "text": "Raita",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 7
        },
        {
            "level": "H1",
            "text": "Ratatouille",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 7
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 8
        },
        {
            "level": "H1",
            "text": "Rau Muong Xao Toi (Stir-Fried Morning Glory with Garlic)",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 8
        },
        {
            "level": "H1",
            "text": "Refried Beans",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 8
        },
        {
            "level": "H1",
            "text": "Rfissa",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 8
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 9
        },
        {
            "level": "H1",
            "text": "Rice and Peas",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 9
        },
        {
            "level": "H1",
            "text": "Rillettes",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 9
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 10
        },
        {
            "level": "H1",
            "text": "Risotto",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 10
        },
        {
            "level": "H1",
            "text": "Rösti (Potato Pancakes)",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 10
        },
        {
            "level": "H1",
            "text": "Rotkohl (Braised Red Cabbage)",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 10
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 11
        },
        {
            "level": "H1",
            "text": "Saganaki",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 11
        },
        {
            "level": "H1",
            "text": "Salade Niçoise",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 11
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 11
        },
        {
            "level": "H1",
            "text": "Salpicão",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 12
        },
        {
            "level": "H1",
            "text": "Saltfish Fritters",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 12
        },
        {
            "level": "H1",
            "text": "Samosa",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 12
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 13
        },
        {
            "level": "H1",
            "text": "Satay",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 13
        },
        {
            "level": "H1",
            "text": "Sauerkraut",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 13
        },
        {
            "level": "H1",
            "text": "Shakshuka",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 13
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 14
        },
        {
            "level": "H1",
            "text": "Shawarma",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 14
        },
        {
            "level": "H1",
            "text": "Skordalia",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 14
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 15
        },
        {
            "level": "H1",
            "text": "Som Tum (Green Papaya Salad)",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 15
        },
        {
            "level": "H1",
            "text": "Sopa de Tortilla",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Ingredients",
            "page": 15
        },
        {
            "level": "H2",
            "text": "Instructions",
            "page": 15
        }
    ]
}